src/rest-server/src/models/v2/job-attempt.js

```diff
@@ -60,9 +61,11 @@ const healthCheck = async () => {
       if (result.statusCode === 200) {
         return true;
       } else {
+        logger.warn(`elastic search return status: ${result.statusCode}`);
```

Comment:
Also log http error msg?

Reply:
there is no message property in elastic search response, maybe status code is enough as a hint

Reply:
But we previously get "index not found" msg, how to get it?

Reply:
Log whole result (and response in other places)